src/NzbDrone.Core/Datastore/Migration/203_release_type.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using Dapper;
+using FluentMigrator;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(203)]
+    public class release_type : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Blocklist").AddColumn("ReleaseType").AsInt32().WithDefaultValue(0);
+            Alter.Table("EpisodeFiles").AddColumn("ReleaseType").AsInt32().WithDefaultValue(0);
+
+            Execute.WithConnection(UpdateEpisodeFiles);
```

Comment:
This only updates episode files to keep it lighter than trying to update existing history and blocklist, though it would be possible.

Reply:
Parsing should be pretty quick, but let me try and benchmark it a bit to see if that'll trigger often.

Reply:
I pulled in a list of 12060 file paths and ran them through the parser, 12045 of them parsed successfully, with 15 failing for whatever reason. Total time taken was 923ms. There will be some additional overhead reading/writing from/to the DB, but I think we still have a lot of overhead here.

Reply:
What's at it in Prowlarr now? Not opposed to increasing the limit, but also, of course it's FreeBSD.

Reply:
I've bumped it to 5 minutes, let's see how that goes.